jvm-packages/dev/build.sh

```diff
@@ -17,5 +17,5 @@ rm /usr/bin/python
 ln -s /opt/rh/python27/root/usr/bin/python /usr/bin/python
 
 # build xgboost
-cd /xgboost/jvm-packages;mvn package
+cd /xgboost/jvm-packages;ulimit -c unlimited;mvn package
```

Comment:
Why `ulimit -c unlimited` here? Did you run into stack space issue?

Reply:
yes, it is a script used in docker container, I found (at least in my environment) that the stacktrace cannot be shown when debugging issues like code crashed in native layer